test/e2e/install/install_test.go

```diff
@@ -214,20 +238,27 @@ func installMayastor() {
 	applyDeployYaml("../test-yamls/moac-deployment.yaml")
 	applyDeployYaml("../test-yamls/mayastor-daemonset.yaml")
 
-	// Given the yamls and the environment described in the test readme,
+	// Given the yaml files and the environment described in the test readme,
 	// we expect mayastor to be running on exactly numMayastorInstances nodes.
-	Eventually(mayastorReadyPodCount,
+	Eventually(func() int {
```

Comment:
Was the old version actually incorrect? I know we use the new style everywhere else, just curious.

Reply:
With the old style I wasn't seeing the diagnostics other than the first one, so not sure what was happening.